oughtinc/ergo
# Retrieve, plot, and sample from Metaculus community prediction distributions

These features are used in [`notebooks/community_distributions_v2.ipynb`](https://github.com/oughtinc/ergo/blob/community-prediction/notebooks/community_distributions_v2.ipynb). You should probably start there to understand what's going on.

Main changes:
- get the community prediction from Metaculus
- graph the community prediction and our prediction
- test this functionality
- various small improvements to make it work, e.g. improve how we convert samples to a dataframe

semi-related changes:
- change how log questions are graphed
- fix some prediction submission stuff with the Metaculus API
- fix sample normalization per correspondence with Metaculus

Unrelated changes:
- add some development tips to the README (sorry, should really have been a separate PR!)

Some things are still a bit rough:
- https://github.com/oughtinc/ergo/issues/62
- https://github.com/oughtinc/ergo/issues/63
- https://github.com/oughtinc/ergo/issues/67
- https://github.com/oughtinc/ergo/issues/64
- https://github.com/oughtinc/ergo/issues/65
- https://github.com/oughtinc/ergo/issues/66

But I think it's probably better to go ahead and merge this and then fix those

(note that I've assigned the last 3 to myself to work on soon)
